src/components/CostsSummary/CostsSummary.jsx

```diff
@@ -0,0 +1,72 @@
+import styles from './CostsSummary.module.css'
+
+const CostsSummary = () => {
+  // These will be replaced by dynamic data fetched from the database in the future
+  const checkInDate = '2024-09-20'
+  const checkOutDate = '2024-09-25'
+  const pricePerNight = 90
+  const airbnbServiceFee = 30
+  const cleaningFee = 60
+  const longStayDiscount = 50
+  const nightsCountForDiscount = 5
+
+  const calculateNights = (checkIn, checkOut) => {
+    return (new Date(checkOut) - new Date(checkIn)) / (1000 * 60 * 60 * 24)
+  }
+
+  const nights =
+    checkInDate && checkOutDate ? calculateNights(checkInDate, checkOutDate) : 0
+  const isDiscount = nights >= nightsCountForDiscount
+
+  const basePrice = nights * pricePerNight
+  const totalPrice =
```

Comment:
It would be great to pass these values to your component as props. You can read about it here https://react.dev/learn/passing-props-to-a-component
Then you can call your component from where you want to display it as `<CostsSummary checkInDate="2024-0-20" pricePerNight=20 ... />`

Reply:
Thank you for the suggestion! I’ll start working on passing these values as props to the component.